MagTraining-Setup-app/build.gradle

```diff
@@ -40,12 +40,14 @@ dependencies {
 
     // MAS Specific Libraries
 //    implementation 'com.ca.mas.core:mas-foundation-release@aar'
-    implementation 'com.ca:mas-foundation:2.1.00'
-    implementation 'com.ca:masui:2.1.00' // (Optional) Only required when using MASUI Template. The MAS UI library provides sample user interfaces for Login, OTP, Social Login, and Enterprise Browser.
-    implementation 'com.ca:mas-storage:2.1.00' // (Optional) Only required when using mas storage
-
+//    implementation 'com.ca:mas-foundation:2.1.00'
```

Comment:
Please remove the unnecessary commented versions - whatever the dependencies are implemented, only keep them.